src/option_parser.c

```diff
@@ -606,7 +607,7 @@ int load_ini_file(FILE *fp)
 
                         g_free(current_section);
                         current_section = (g_strdup(start + 1));
-                        new_section(current_section);
+                        LOG_D("[%s]", current_section);
```

Comment:
Pretty print the section to make it look like in the ini file.

Reply:
Also, do not call `new_section()` here, because the later call to `add_entry()` will simply create it, if it does not exist already.